universitetet-i-oslo-rettsvitenskap.csl

```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="nb-NO" demote-non-dropping-particle="display-and-sort" page-range-format="expanded">
+  <info>
+    <title>Universitetet i Oslo - Rettsvitenskap (Norwegian - Bokmål)</title>
+    <title-short>UiO-rettsvitenskap</title-short>
+    <id>http://www.zotero.org/styles/universitetet-i-oslo-rettsvitenskap</id>
+    <link href="http://www.zotero.org/styles/universitetet-i-oslo-rettsvitenskap" rel="self"/>
+    <link href="http://www.ub.uio.no/skrive-publisere/referere/referansestiler/rettsvitenskap.html" rel="documentation"/>
+    <author>
+      <name>Hans Gunnar Slokvik Lian</name>
+      <email>h.g.s.lian@ub.uio.no</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <summary>University of Oslo Law Library's citation style. Created for use primarily with Norwegian legal sources.</summary>
+    <updated>2017-11-16T10:04:01+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="nb-NO">
+    <terms>
+      <term name="editor" form="verb-short">red.</term>
+      <term name="translator" form="verb-short">overs.</term>
+      <term name="translator" form="short">overs.</term>
+      <term name="editortranslator" form="verb-short">
+        <single>red. og overs.</single>
+        <multiple>red. og overs.</multiple>
+      </term>
+      <term name="editortranslator" form="verb">
+        <single>Redigert og oversatt av</single>
+        <multiple>Redigert og oversatt av</multiple>
+      </term>
+      <term name="translator" form="short">overs.</term>
+      <term name="and others"> </term>
```

Comment:
@hgsl, did you test whether you get the correct output if you use an empty term here? (i.e. `<term name="and others"></term>` instead of `<term name="and others"> </term>`)

Reply:
@adam3smith, other that this I have no comments.

Reply:
It appears to yield the same output, so I'll use the empty term instead - thanks!